include/convex_bodies/hpolytope.h

```diff
@@ -324,8 +326,13 @@ class HPolytope{
         return std::pair<NT, NT>(min_plus, max_minus);
     }
 
-    std::pair<NT,NT> line_intersect(Point &r, Point &v, VT& Ar,
-            VT& Av, const NT &lambda_prev, bool pos = false) {
+    std::pair<NT,NT> line_intersect(Point &r,
```

Comment:
why not const reference to r and v points? (This comment goes for all the other classes of convex bodies)